data/linamiko.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Lina",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "**Career Chameleon** - constantly looking for new skills to learn and opportunities | Firm believer that **mindset is key**.",
+  "avatar": "https://github.com/linamiko.png",
+  "links" [
```

Comment:
Missing `:` after `links` on line 7
It should be `"links" : [ `

Reply:
Ah thanks so much! Updating it now.